### PR TITLE
Update recs and add some files for ease of getting started

### PR DESCRIPTION
**Pull request by luketlancaster**

### CODE REVIEW

**Comment by luketlancaster** [comment]
closes #1

**Comment by michaelclark2** [comment]
i would add a step 5 for running `hs` in the src directory, that came back to bite me when looking for my console log in the first Check you work section
that being said, i got some errors when running the code at the first **Check your work** section
`Uncaught SyntaxError: The requested module './Clinic.js' does not provide an export named 'default'`

adding the `export default {};` to clinic.js fixed it, but thats the next step.  maybe commenting out the import in main.js?

also also, the `Person is tested` test wont pass without the `clinic.diagnose()` function, so the instructions are misleading, we could remove the clinic.diagnose call on line 9 in patient.testing.test.js

File: README.md [hunk 1]
```diff
@@ -81,8 +79,8 @@ Your next task is to create HTML representations of the people who have been tes
 ### Create Patient HTML
 
 1. Create a `scripts/PatientList.js` module.
-1. Define and export a `PatientList` function.
-1. The `PatientList` function must import that array of patients from the from the `Clinic.js` module.
+1. In the `PatientList` module define and export a `PatientList` function.
```
Comment (michaelclark2) [inline README.md]:
```suggestion
1. In the `PatientList` module define and export a `patientList` function.
```

File: README.md [hunk 2]
```diff
@@ -81,8 +79,8 @@ Your next task is to create HTML representations of the people who have been tes
...
+1. The `PatientList` function must get the array of patients from the from the `Clinic.js` module.
```
Comment (michaelclark2) [inline README.md]:
```suggestion
1. The `patientList` function must get the array of patients from the from the `Clinic.js` module.
```

File: README.md (hunk 1 above)
Comment (michaelclark2) [reply README.md]: the test is looking for a `patientList` function

File: README.md [hunk 3]
```diff
@@ -98,13 +96,13 @@ The resulting HTML should look like the following example. Recall the `${}` synt
         </div>
     </section>
     ```
-1. The function must put all of the HTML representations into a single string. Recall that there are ways to turn an array's values into a single string.
+1. The function must put all of the HTML representations into a single domString. Recall that there are ways to turn an array's values into a single string.
 1. The function then must return that single string that has all of the patient HTML representation in it.
-1. In the `main.js` module, invoke the `PatientList` component function. Take the string of HTML representations that it returns and update the inner HTML of the article element you created in `index.html`. You need to remember how to get a reference to DOM element in JavaScript and then assign it some HTML.
+1. In the `main.js` module, invoke the `PatientList` component function. Take the string of HTML representations that it returns and update the inner HTML of the article element you created in `index.html`. You need to remember how to get a reference to DOM element in vanilla JavaScript (no jQuery!!!!) and then assign it some HTML.
```
Comment (michaelclark2) [inline README.md]:
```suggestion
1. In the `main.js` module, invoke the `patientList` component function. Take the string of HTML representations that it returns and update the inner HTML of the article element you created in `index.html`. You need to remember how to get a reference to DOM element in vanilla JavaScript (no jQuery!!!!) and then assign it some HTML.
```

**Review by michaelclark2 — COMMENTED** [review]
the function should be `patientList` not `PatientList`

**Comment by luketlancaster** [comment]
take another look @michaelclark2, if you would. Updated the readme with your requested changes and made the `patient.testing.test` a little smarter/more specific. Also made specific commands to run each test suite, so that the output is a little more sane

**Pull request merged**